hnn_core/viz.py

```diff
@@ -233,6 +233,12 @@ def plot_dipole(dpl, tmin=None, tmax=None, ax=None, layer='agg', decim=None,
         ints can be provided. These are applied successively.
     color : tuple of float
         RGBA value to use for plotting (optional)
+    average : bool
+        If True, render the average across all dpls.
+    alpha : float
+        The alpha value of individual dpls.
+    linewidth : float
+        The width of dpl lines.
```

Comment:
let's not expose these. The more you expose parameters, the more you have to maintain.